app/src/main/res/layout/fragment_dashboard_plus.xml

```diff
@@ -170,7 +170,7 @@
                     app:layout_constraintEnd_toEndOf="parent"
                     app:layout_constraintStart_toStartOf="parent"
                     app:layout_constraintTop_toBottomOf="@id/dash_location_off"
-                    app:visibleOrGone="@{vm.exposureNotificationsEnabled}" />
+                    app:visibleOrGone="@{false}"/>
```

Comment:
@davidvavra toto je ok? fungovat to funguje. akurat neviem ci sa to nebude overridovat s nastavenim v kode. mozno by sa mala ta visibility vyhodit prec z tejto karty?

Reply:
hodil bych tam android:visibility="gone", je to rychlejší než binding a v kódu se to pak stejně mění programově